### PR TITLE
Remove username from edit user form

### PR DESCRIPTION
**Pull request by martensonbj**
As a registered user you can no longer edit your username. This prevents duplications down the road and issues with slugs in the URL. Usernames should be unique and permanent for each user. I also added a test to verify all this was working. Username is still visible on create new user view of form.

### CODE REVIEW

**Comment by martensonbj** [comment]
Also accidentally made changes to the mobile navbar in this branch, so card 66 is also wrapped up in this and ready to go. Side nav bar is now a slightly opaque black with lighter colored font. Changed the link "Logged in as Brenna" to "Dashboard (Brenna)" so the link makes more sense as to where you are directed.

**Comment by ToniRib** [comment]
New mobile navbar looks SOOO much better! Love the transparency.

Also, I checked out the edit functionality on user, artist, and admin and it all seems to work. Good job!

File: app/views/users/_form.html.erb [hunk 1]
```diff
@@ -28,9 +28,11 @@
   <%= f.label :zipcode %>
   <%= f.text_field :zipcode %>
   <br>
-
-  <%= f.label :username %>
-  <%= f.text_field :username  %>
+  
+  <% if @user != current_user %>
```
Comment (ToniRib) [inline app/views/users/_form.html.erb]: @martensonbj couldn't we also do <% unless current_user %> to just check if someone is signed in? You can't get to this page unless you're the current user anyway.

Comment (martensonbj) [inline app/views/users/_form.html.erb]: Makes sense! Im ok with that

**Comment by ToniRib** [comment]
I'm going to try merging the latest master into this so we can close it.

**Pull request merged**